addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/RequestorJob.java

```diff
@@ -62,7 +62,11 @@ public RequestorJob(HttpSender httpSender) {
     }
 
     @Override
-    public void verifyJobSpecificData(LinkedHashMap<?, ?> jobData, AutomationProgress progress) {
+    public void verifyJobSpecificData(AutomationProgress progress) {
+        LinkedHashMap<?, ?> jobData = this.getJobData();
+        if (jobData == null) {
+            return;
+        }
```

Comment:
Should the other jobs do the same?

Reply:
Depends if they require it ;) I'll check them all...

Reply:
Ah no, it cant be null :)